Domain.Sql/StorableEvent.cs

```diff
@@ -21,12 +21,13 @@ public StorableEvent()
             Timestamp = DateTimeOffset.UtcNow;
         }
 
+        [Index(IsUnique = true), Index("IX_Id_StreamName_Type", 1, IsUnique = true)]
         public long Id { get; set; }
 
-        [MaxLength(255)]
+        [MaxLength(50), Index("IX_Id_StreamName_Type", 2, IsUnique = true)]
```

Comment:
The lengths are intended to be small enough to actually support this index. Previously, they were too large.

As for the defaults, I think that it's reasonable as documentation of the values, since the lifetime of the database will span EF versions and the EF defaults are not explicit.